.eslintrc.json

```diff
@@ -694,7 +694,6 @@
         "jsdoc/check-values": "warn",
         "jsdoc/empty-tags": "warn",
         "jsdoc/implements-on-classes": "warn",
-        "jsdoc/newline-after-description": ["warn", "always"],
```

Comment:
https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v42.0.0

The noted replacements did not work for me as expected so I just removed the rule since I did not consider it that important.